__tests__/app.test.js

```diff
@@ -103,4 +106,39 @@ describe("GET /api/topics", () => {
     });
     
     
-  })  
\ No newline at end of file
+  })  
+
+  describe("when given a valid article ID returns with comments for correct artice", () => {
+    it("returns an array of comments objects with correct properties", () => {
+      return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({body}) => {
+        expect(body).toBeInstanceOf(Array);
+        expect(body).toBeSorted({ descending: true });
+        expect(body.length).toBeGreaterThan(0);
+        expect(typeof body[0]).toBe("object");
```

Comment:
Should have a forEach loop here to check all of the comments pass these expectations